tests/testthat/adtteSpec/.gitignore

```diff
@@ -0,0 +1,2 @@
+# {shinytest2}: Ignore new debug snapshots for `$expect_values()`
+*_.new.png
```

Comment:
could this be ignored on higher-level `.gitignore` file ? so that we don't have to repeat for every test?